Assets/Plugins/iOS/Shopify/BuyTests/PaymentSessionTests.swift

```diff
@@ -68,7 +68,7 @@ class PaymentSessionTests: XCTestCase {
     //  MARK: - Supported Networks -
     //
     func testSupportedNetworks() {
-        XCTAssertEqual(PaymentSession.supportedNetworks, [.amex, .masterCard, .visa, .discover])
```

Comment:
No I suppose it could of went into a different PR, but it's kinda small, just shoved it in here.
Yes it will be dynamic soon.